### PR TITLE
Fix upgrade and build

### PR DESCRIPTION
**Pull request by arnaud-morvan**

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/env.project [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Custom project values
+TILEGENERATION_SQS_QUEUE=<queue_name>
+TILEGENERATION_S3_BUCKET=<bucket_name>
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/env.project]: Shouldn't we put the Postgres configuration in this file?

File: scripts/upgrade [hunk 2]
```diff
@@ -40,15 +40,15 @@ parser.add_argument("step", nargs="?", help="the step to run")
 parser_finalise = argparse.ArgumentParser(description="Upgrade the project", add_help=False)
 parser_finalise.add_argument("--finalise", action="store_true", help="finalise the upgrade")
 parser_finalise.add_argument("--help", action="help", help="show this help message and exit")
-parser_finalise.add_argument("build_arg", nargs="*", help="a build argument")
+parser_finalise.add_argument("build_args", nargs="*", help="build arguments ex: -- --pgschema main env.default env.project")
```
Comment (sbrunner) [inline scripts/upgrade]: `build_arg` is better for the --help ...
=>
[build_arg [build_arg ...]]

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl [hunk 3]
```diff
@@ -22,9 +22,9 @@ parser = argparse.ArgumentParser(description="Build the project")
 parser.add_argument("--verbose", help="Display the docker build commands")
 parser.add_argument("--config", action="store_true", help="Build only the configuration image")
 parser.add_argument("--geoportal", action="store_true", help="Build only the geoportal image")
-parser.add_argument("--pgschema", default="main", help="The main schema name used in PostgreSQL")
+parser.add_argument("--pgschema", required=True, help="The main schema name used in PostgreSQL")
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl]: Shouldn't we get the PGSCHEMA from the env files?

**Comment by arnaud-morvan** [comment]
@sbrunner : I've refactored a little the build script (added a main method).
Now schema is taken from environement and if env_files are not provided, it reuse the previous ones.
I also added some lines in local.mk to be able to test my work.

**Comment by sbrunner** [comment]
You can ignore the Pull request title :-)

File: geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl [hunk 4]
```diff
@@ -18,86 +18,99 @@ def call(verbose, arguments, **kwargs):
...
-        f.write(result.read().decode())
-    os.chmod("upgrade", os.stat("upgrade").st_mode | stat.S_IXUSR)
-    try:
-        subprocess.check_call(["./upgrade", full_version])
-    except subprocess.CalledProcessError:
-        sys.exit(1)
-    sys.exit(0)
+        sys.exit(0)
+
+    env_files = args.env_files or load_env(['.env'])['ENV_FILES'].split(" ")
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl]: Why the or ...?

Comment (arnaud-morvan) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl]: This is a default value
if args.env_files != [] => args.env_files
else load_env(['.env'])['ENV_FILES'].split(" ")

Comment (arnaud-morvan) [reply geoportal/c2cgeoportal_geoportal/scaffolds/create/build_tmpl]: if user do not give any env_files, use previous ones.

**Comment by sbrunner** [comment]
@arnaud-morvan fixed with a new commit, For me we can merge it as it :-)

**Comment by arnaud-morvan** [comment]
@sbrunner : merged, thanks for the fix

**Pull request merged**